.github/workflows/check-test-release.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   MLEM_TESTS: "true"
+  MLEM_DEBUG: "true"
```

Comment:
Another bug:

```shell
$ mlem --tb deployment run heroku app.mlem --model models/rf --app_name mlem-get-started
...
mlem.core.errors.MlemObjectNotFound: MLEM object was not found at `heroku`
```
This should work, judging by your examples [here](https://github.com/iterative/mlem/pull/417#issue-1383952509).

Reply:
I don't face this issue, I am using
`mlem --tb deployment run heroku app.mlem --model sk-model --app_name mlem-get-started`
and I face the issue of `FileNotFoundError: [Errno 2] No such file or directory: 'heroku'`
which is expected and is because of me not passing the API key which it tries to determine using
`subprocess.check_output(["heroku", "auth:token"])`
and I don't have the `heroku` CLI installed so...

Can you try this again @aguschin? maybe make sure you are using the `release 0.3` branch, do a
`pip install -e .` again after switching to it.

Reply:
FIxed! Thanks